scripts/lib/setup/installers.sh

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env bash
+
+function install_node() {
+  if nvm_installed; then
+    install_node_via_nvm
+  else
+    install_node_via_package_manager
+  fi
+}
+
+function install_and_setup_package_manager() {
+  if is_macos; then
+    install_homebrew_if_needed
+    brew_tap "caskroom/cask"
+  elif is_linux; then
+    # Linux
+    buildtools=(
+      autoconf
+      automake
+      build-essential
+      cmake
+      curl
+      libssl-dev
+      libtool
+      make
+      python-dev
+      wget
+      unzip
```

Comment:
Add `g++` here. Seems to be a requirement on AWS Ubuntu 16.04 machines. For some reason `build-essential` doesn't seem to cover the bases required for `lzma-native` in particular.

Reply:
Done